docs/release-notes/13-2-0/README.md

```diff
@@ -3,22 +3,87 @@ title: OpenProject 13.2.0
 sidebar_navigation:
     title: 13.2.0
 release_version: 13.2.0
-release_date: TBD
+release_date: 2023-01-17
 ---
 
 # OpenProject 13.2.0
 
-Release date: TBD
+Release date: 2023-01-17
 
 We released [OpenProject 13.2.0](https://community.openproject.com/versions/1979).
-The release contains several bug fixes and we recommend updating to the newest version.
 
+It brings improvements for the **OneDrive/SharePoint integration**, now also allowing **Manual project folders**. In addition to that, both OneDrive/SharePoint and Nextcloud integrations were improved by **showing deleted files in OpenProject work packages**.
```

Comment:
Suggestion: "Along other features, this release brings improvements..."
--> so that the reader knows right away, there is more